### PR TITLE
Feature/home screen

### PR DESCRIPTION
**Pull request by ioanagirbap33**

### CODE REVIEW

File: MarketMatrix/src/components/HomeScreenComponents/TotalValueLineChart.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+export const TotalValueLineChart = ({currency}: TotalValueLineChartProps) => {
+  const [user, setUser] = useState<UserData[]>([]);
+  const [changes, setChanges] = useState<StockChanges[]>([]);
+
+  let chartValues: number[] = [];
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await axios.get('http://localhost:8081/src/db.json');
```
Comment (roxanap33) [inline MarketMatrix/src/components/HomeScreenComponents/TotalValueLineChart.tsx:19]: You can create a separate function and move it to http directory and then use it here

File: MarketMatrix/src/components/chart/ChartConfiguration.tsx [hunk 2]
```diff
@@ -16,28 +17,66 @@ export const ChartConfiguration = ({
   const chartConfig = {
     chart: {
       type: chartType,
+      // backgroundColor: '#ffffff',
```
Comment (roxanap33) [inline MarketMatrix/src/components/chart/ChartConfiguration.tsx:20]: This is unnecessary

File: MarketMatrix/src/components/chart/ChartConfiguration.tsx [hunk 3]
```diff
@@ -16,28 +17,66 @@ export const ChartConfiguration = ({
...
       line: {
-        dataLabels: {enabled: true},
+        dataLabels: {
+          enabled: true,
+          style: {
+            fontSize: 26,
+          },
+        },
+        tooltip: {
+          headerFormat: '<span style="font-size: 30px">{point.key}</span><br/>', // Set font size of tooltip header
```
Comment (roxanap33) [inline MarketMatrix/src/components/chart/ChartConfiguration.tsx:67]: Do you want those comments there?

File: MarketMatrix/src/constants/Interfaces.ts [hunk 4]
```diff
@@ -1,3 +1,20 @@
...
+  image: string;
+  ceo: string;
+  city: string;
+  state: string;
+  industry: string;
+  sector: string;
+  type: string;
+}
+
 // Add global interfaces
```
Comment (adrianacostep33) [inline MarketMatrix/src/constants/Interfaces.ts:18]: Please delete this comment if you don't need it

**Review by roxanap33 — APPROVED** [review]

**Review by adrianacostep33 — APPROVED** [review]
Looks good

**Pull request merged**